src/rdiff_backup/Globals.py

```diff
@@ -20,10 +20,14 @@
 
 import re
 import os
-from . import Version
 
 # The current version of rdiff-backup
-version = Version.version
+# Get it from package info or fall back to DEV version.
+try:
+    import pkg_resources
```

Comment:
That implies a dependency on Python's setuptools at runtime, correct? This should be at least documented in the README.md

Reply:
Not sure what you want me to add to the README.md. I assume setuptools to be installed in all the possible scenario: 
1. install from pip: pip depends on setuptools
2. install from `python setup.py install`: setup.py already depends on `setuptools`
3. install from a deb or rpm: setuptools should to added as a dependencies in the deb or rpm

Reply:
Seen like this...